tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -1296,6 +1296,10 @@ tf_module {
     name: "gather_nd"
     argspec: "args=[\'params\', \'indices\', \'name\', \'batch_dims\'], varargs=None, keywords=None, defaults=[\'None\', \'0\'], "
   }
+  member_method {
+    name: "generate_bounding_box_proposals"
```

Comment:
Probably belongs in tf.image, not tf

Reply:
Implementation is in tf.image. This is another auto-generated file. What kind of changes should I do so that api generator puts it tf.image?

Reply:
It should be done now.